samwise.js

```diff
@@ -10,18 +10,16 @@
     "use strict";
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.